js/app.js

```diff
@@ -5,10 +5,25 @@
  * and export dependency modules to any new scripts
  * you create.
  */
-
+import Player from './classes/player.js';
+import Enemy from './classes/enemy.js';
+import './components/touch-icon.js';
+import InfoSection from './components/info-section.js';
 import Rand from "./utils/rng.js";
-import Player from "./classes/player.js";
-import Enemy from "./classes/enemy.js";
+
+// KEVIN TO TEST ICON CLASS
+const Chad = new Player('Chad',{hp:100, atk:5, spd:5, def:0});
+const Delilah = new Enemy('skeleton soldier',{hp:10, atk:3, spd:3, isLocked:false});
```

Comment:
this doesn't need to be changed right now, but I'm stating my concern on a comment that way I can remember to come back to this later on in the future: 

I'm concerned that the isLocked property is less an Enemy/Item concern and more an icon bar concern. I think once the rooms start generating these things, they can take care of giving touch-icon the properties they need. For now, as long as it works we're fine.

Reply:
That can definitely work. And I see the point is separating that out as it really isn't a stat.